pages/resources.js

```diff
@@ -0,0 +1,150 @@
+import React, {Component} from "react";
+import styled from "styled-components";
+
+import PageTemplate from "../components/templates/PageTemplate";
+import IntroHero from "../components/molecules/IntroHero";
+import Card from "../components/molecules/Card";
+import CardGroup from "../components/molecules/CardGroup";
+import SearchBar from "../components/molecules/SearchBar";
+
+import Container from "../components/templates/Container";
+
+import {fetchResources} from '../lib/fetchResources';
+import {categories} from '../config.json';
+import Dropdown from "../components/molecules/Dropdown";
+
+const ResourceFilters = styled("div")`
+    padding-bottom: 25px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 15px;
+    grid-row-gap: 15px;
+`;
+
+export default class Resources extends Component {
+    state = {
+        resources: this.props.resources,
+        constantResources: this.props.resources,
+        filterResources: "Show All",
+        filterPrice: "Show All",
+        status: "Loading..."
+    };
+
+    searchResources = (event) => {
+        const query = event.target.value.toLowerCase();
+
+        if (query === "") {
+            this.setState({resources: this.state.constantResources});
+        } else {
+            let resourcesThatMatchQuery = [];
+
+            for (const resource of this.state.constantResources) {
+                if (resource.name.toLowerCase().includes(query)) {
+                    resourcesThatMatchQuery.push(resource);
+                }
+            }
+
+            if (!resourcesThatMatchQuery.length) {
+                this.setState({status: "No matches found..."});
+            }
+
+            this.setState({
+                status: null,
+                resources: resourcesThatMatchQuery
+            });
+        }
+    };
+
+    filterResources = async (event) => {
+        let resources = this.state.constantResources;
+        const value = event.target.value;
+
+        if (value !== "Show All") {
+            resources = await resources.filter((resource) => resource.category.toLowerCase() === value);
+        }
+
+        this.setState({resources, filterResources: value});
+    };
+
+    filterPrice = async (event) => {
+        let resources = this.state.constantResources;
+        const value = event.target.value;
+
+        if (value !== "Show All" && ["true", "false"].includes(value)) {
+            resources = await resources.filter((resource) => resource.free === JSON.parse(value));
+        }
+
+        this.setState({resources, filterPrice: value});
+    };
+
+    render() {
+        const {resources, status} = this.state;
+
+        return (
+            <PageTemplate page="Resources">
```

Comment:
It would be really neat to take advantage of the custom meta stuff we can do with Next:
```suggestion
            <PageTemplate page="Resources" meta={{
                description: "A collection of programming resources curated by the CodeSupport community."
            }}>
```